### DISCUSSION TITLE
Marvell Armada 88F6828 dual-core 1.3GHz

### DISCUSSION THREAD

**Discussion by skipdashu** [bb-qq/r8152#363] [Q&A] [answered]
This is the CPU in my old 416j... I've upgraded it to DSM 7.2.1-69057 Update 3.  

Is there a DSM 7.2 driver for this CPU so I can use another Wavlink USB C to Ethernet Adapter 2.5 Gbps, 2-in-1 USB to 2.5G Ethernet Adapter (same as I successfully used on DS224+) ?  I'm not finding one and the DSM 7,1 version is rejected in package manager.

**Comment by skipdashu** [comment]
Where do I do my begging at?  ;-)

**Reply by bb-qq** [reply, marked as answer]
DS416j uses [Marvell Armada 388 88F6828](https://kb.synology.com/en-global/DSM/tutorial/What_kind_of_CPU_does_my_NAS_have#x16-series), so you can use this driver:
https://github.com/bb-qq/r8152/releases/download/2.17.1/r8152-armada38x-2.17.1-1_7.1.spk

**Reply by bb-qq** [reply]
I think I have answered your question, so I will close this thread. If you have any questions, feel free to re-open it.
